hyperdash/constants.py

```diff
@@ -45,7 +45,7 @@ def get_hyperdash_logs_home_path():
 
 
 def get_hyperdash_logs_home_path_for_job(job):
-    return os.path.join(get_hyperdash_logs_home_path(), slugify(job))
+    return os.path.join(get_hyperdash_logs_home_path(), slugify(unicode(job)))
```

Comment:
would be good to figure out what changed that now requires the call to unicode just to be sure we're not breaking anything.

Reply:
The call is in response to this issue that appears in python 3:
```
  File "/Users/andrew/git/hyperdash-sdk-py/hyperdash/constants.py", line 54, in get_hyperdash_logs_home_path_for_job
    return os.path.join(get_hyperdash_logs_home_path(), slugify(job))
  File "/Users/andrew/anaconda3/lib/python2.7/site-packages/slugify.py", line 26, in slugify
    unicodedata.normalize('NFKD', string)
TypeError: normalize() argument 2 must be unicode, not str
```
Perhaps upstream the job name used to be converted to unicode, but is no longer.